manage_quay/master.adoc

```diff
@@ -105,7 +105,7 @@ include::modules/metrics-authentication.adoc[leveloffset=+3]
 //include::modules/proc_manage-quay-geo-replication.adoc[leveloffset=+1]
```

Comment:
Here only show the autoprune policy configurations of Quay Organization, maybe need to add Pic to demonstrate the  autoprune policy configurations for user namespace.